src/components/vm/disks/vmDiskColumns.tsx

```diff
@@ -87,10 +88,33 @@ export const DiskSourceCell = ({
     const chunks: React.ReactNode[] = [];
     DISK_SOURCE_LIST.forEach(entry => addOptional(chunks, getDiskSourceValue(diskSource, entry.name), entry.name, entry.label));
 
-    return <DescriptionList isHorizontal>{chunks}</DescriptionList>;
+    return chunks;
 };
 
-export const DiskExtras = ({
+export const DiskSourceAbbrev = ({
+    disk,
+} : {
+    disk: VMDisk,
+}) => {
+    const diskSource = disk.source;
+    if (diskSource.file) {
+        return basename(diskSource.file);
+    } else if (diskSource.dev) {
+        return diskSource.dev;
+    } else if (diskSource.volume) {
+        return diskSource.volume;
+    } else if (diskSource.host.name && diskSource.host.port) {
+        return diskSource.host.name + ":" + diskSource.host.port;
+    } else if (diskSource.host.name) {
+        return diskSource.host.name;
+    } else if (disk.device == "cdrom") {
+        return <div className="pf-v6-u-text-color-subtle">{_("(empty)")}</div>;
+    } else {
+        return null;
```

Comment:
This added line is not executed by any test.  [Details](https://logs-cockpit.apps.ocp.cloud.ci.centos.org/pull-2324-02ce13bf-20250922-141831-fedora-42-devel/Coverage/lcov/github-pr.diff.gcov.html)